.github/workflows/e2e-pipeline.yml

```
@@ -0,0 +1,477 @@
+          echo "Run ID: ${{ github.run_id }}"
+          echo ""
+          echo "📊 Test artifacts uploaded to:"
+          echo "${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
+
+  cleanup-server:
+    needs: [e2e-tests]
+    runs-on: self-hosted
+    if: always()
+    steps:
+      - name: Stop Cypress Servers
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:3:8: Double quote to prevent globbing and word splitting

(shellcheck)